src/en/guide/theWebLayer/interceptors/interceptorOrdering.adoc

```diff
@@ -27,11 +27,11 @@ int order = HIGHEST_PRECEDENCE + 50
 int order = LOWEST_PRECEDENCE - 50
 ----
 
-To find out the computed order of interceptors you can add a debug logger to `logback.groovy` as follows:
+To find out the computed order of interceptors you can add a debug logger to `logback-spring.xml` as follows:
```

Comment:
This also applies to `logback.xml`. Should we re-phrase to '... add a debug logger to your logback configuration as follows:'?

Reply:
Same here, in my opinion we should not document `logback.xml`. Instead we could specify the whole location like:

> to `grails-app/conf/logback-spring.xml` as follows